webhooks/user_webhook.go

```diff
@@ -27,6 +29,21 @@ type UserValidator struct {
 func (v *UserValidator) Handle(ctx context.Context, req admission.Request) admission.Response {
 	log := log.FromContext(ctx).WithName("webhook.validate-users.appuio.io")
 
+	// Allow the user to create or update itself.
+	// A special permission, used for controllers, `create control-api.rbac.appuio.io users` can override this.
```

Comment:
I don't quite get where or how this special permission is handled? Is this some form of standard RBAC that I just don't know about?

Reply:
`control-api.rbac.appuio.io` should be `rbac.appuio.io` in the comment 🤦‍♀️